git-workflows/bin/git-workflow.sh

```diff
@@ -230,6 +263,25 @@ delete_branch() {
   git push origin :${BRANCH} 2>&1 | formatOutput
 }
 
+delete_branch_multidir() {
+  log "--- DELETE BRANCH (multidir)---"
+  changedirOrExit "${WORKSPACE}/${REPO_NAME}"
+  git checkout main 2>&1 | formatOutput
+
+  log "removing ${NAMESPACE} from argocd/applicationset.yaml"
+  yq -i "del(.spec.generators[0].list.elements[] | select(.cluster == \"${NAMESPACE}\"))" argocd/applicationset.yaml
+  log "copy '${WORKSPACE}/${REPO_NAME}/argocd/applicationset.yaml' to '${WORKSPACE}/application.yaml'"
+  cp "${WORKSPACE}/${REPO_NAME}/argocd/applicationset.yaml" "${WORKSPACE}/application.yaml"
```

Comment:
Achtung, benötigt Anpassung in gepaplexx-cicd/create-delete workflow für Kompatibilität multi-branch und multidir